.github/workflows/weekly_tests.yaml

```diff
@@ -33,13 +33,17 @@ jobs:
           - charm-storage-connector
           - charm-sysconfig
           - charm-userdir-ldap
+          - charmed-openstack-exporter-snap
           - charmed-openstack-upgrader
+          - dcgm-snap
+          - hardware-observer-operator
```

Comment:
hardware-observer-operator is already in the matrix here - see the include clause below. :)

Reply:
Thanks for catching that